### PR TITLE
Position, maximise and minimise of dlg's and sdg's fixes #1014 fixes #1695 fixes #1724

### LINKED ISSUE #1014: Start position for subdialogs
@africanmathsinitiative/instat  i am just wondering what position would be good for subdialogs start position. 
would it not be nice if it starts on the center of the parent dialog?

### LINKED ISSUE #1695: Double clicking on the title of some dialogs maximises it
This seems to happen for dialogs that have the small (X) in the top right corner e.g. Calculator but not for dialogs with the bigger (X) like regular sequence. @maxwellfundi Can you look into this? But not an urgent task.

### LINKED ISSUE #1724: Fixing for maximise and minimise box on organise and model menus
Hi @dannyparsons , @stevekogo  @maxwellfundi  @stevenndungu ,

I am currently working on this dialogs.

I will let you know when am done.

Thanks

### PR DESCRIPTION
**Pull request by FrancoisJRenaud**
Went through all dialogues and subdialogues and checked the options minimise/maximisebox and start position... For subdialogues set to centreParent. Although are we sure that parent is the main dlg ? Anyway here it is. Let's hope there are no conflicts !!

### CODE REVIEW

**Comment by FrancoisJRenaud** [comment]
shit, conflicts. I m sorry... aahhh

**Comment by FrancoisJRenaud** [comment]
could just undo changes in that file ?

**Comment by dannyparsons** [comment]
I think there's too many changes here for me to confidently accept. It's only visible dialogs that need to be sorted out. So if those are ok, that's fine for now.

**Comment by FrancoisJRenaud** [comment]
Noooo ! I ve just spent 1,5h on this! There are visible dialogues within
the changes! I ve jus systematically set minbox and maxbox to false and set
the start position...

**Comment by dannyparsons** [comment]
Ok I will have to spend some time checking through this. In future these need to be done in much smaller pull requests.
What do we do about the conflict?

File: instat/dlgAdvanceOptions.designer.vb [hunk 1]
```diff
@@ -187,7 +187,10 @@ Partial Class dlgAdvanceOptions
         Me.Controls.Add(Me.chkConvert)
         Me.Controls.Add(Me.chkMessages)
         Me.FormBorderStyle = System.Windows.Forms.FormBorderStyle.FixedToolWindow
+        Me.MaximizeBox = False
+        Me.MinimizeBox = False
         Me.Name = "dlgAdvanceOptions"
+        Me.StartPosition = System.Windows.Forms.FormStartPosition.CenterScreen
```
Comment (dannyparsons) [inline instat/dlgAdvanceOptions.designer.vb]: Sorry I'm thinking of sub dialogs.

**Comment by FrancoisJRenaud** [comment]
Sorry one of my messages didn't go through I think. about conflict, could just exclude that file ?

**Comment by dannyparsons** [comment]
I can't exclude it, I can try to resolve the conflicts

**Comment by FrancoisJRenaud** [comment]
Ah sorry for that. If necessary I can do it on my side. Wil ask John how to do it.

**Pull request merged**